### PR TITLE
optimize galera playbook

### PR DESCRIPTION
**Pull request by jdextercsc**
removed handler and template files that were no longer being used.

### CODE REVIEW

**Comment by mtnbikenc** [comment]
Since the unused handlers are being removed, also remove the commented `notify` actions from tasks/galera.yml.  And any other commented sections that no longer apply.

**Comment by jcpowermac** [comment]
Code review :+1:

File: roles/galera/tasks/galera.yml [hunk 1]
```diff
@@ -29,8 +29,7 @@
   tags: mysql
 
 - name: create clustercheck database user
-  #mysql_user: name={{ clustercheck_db_user }} host=localhost password="{{ clustercheck_db_pass }} state=present
-  #run_once: true
+
```
Comment (mtnbikenc) [inline roles/galera/tasks/galera.yml]: Yank this empty line and the extra lines below, please.

**Comment by mtnbikenc** [comment]
Code review :+1:

**Comment by jcpowermac** [comment]
Code review :+1:

**Pull request merged**